pkg/kademlia/kademlia.go

```diff
@@ -5,243 +5,170 @@ package kademlia
 
 import (
 	"context"
-	"crypto/rand"
+	"errors"
 	"fmt"
-	"log"
-	"net"
-	"strconv"
 
-	bkad "github.com/coyle/kademlia"
 	"github.com/zeebo/errs"
 
 	"storj.io/storj/pkg/dht"
+	"storj.io/storj/pkg/node"
+	"storj.io/storj/pkg/provider"
 	proto "storj.io/storj/protos/overlay"
 )
 
+const (
+	alpha                       = 5
+	defaultIDLength             = 256
+	defaultBucketSize           = 20
+	defaultReplacementCacheSize = 5
```

Comment:
these could be flags?

Reply:
agreed. I'll do it in another PR

Reply:
https://storjlabs.atlassian.net/browse/V3-434